src/Geopilot.Frontend/src/auth/oidcContainerContext.tsx

```diff
@@ -13,13 +13,23 @@ export const OidcContainerProvider: FC<PropsWithChildren> = ({ children }) => {
   const { clientSettings } = useAppSettings();
   const apiSetting = useApiAuthConfiguration();
 
+  const scopes = useMemo(() => {
```

Comment:
Q: Die Client settings unterstützten bereits ein Überschreiben/Erweitern der default scopes. Weshalb brauchen wir neu noch einen API Endpoint dafür?

Reply:
geopilot arbeitet aktuell so, dass wir die eigentlich hardcoded assets überschreiben können. So funktiniert auch das umbenen von logos in png/svg formaten oder dem definieren des angezeigten instanznamens.

Dies würde im übrigen auch ein ersetzten des index.html erlauben, um dort SEO zu betreiben. 

Relevante Codezeile: https://github.com/GeoWerkstatt/geopilot/blob/d165640d4693664cb8890317d0ce9c2cd2cff332/docker-entrypoint.sh#L24

Reply:
Die Scopes für den Auth Request sind noch der einzige Aspekt welche im _client-settings.json_ konfiguriert werden. Ich fänds schön, wenn wir diese dort rausnehmen könnten und - wie [alle anderen Auth Infos](https://github.com/GeoWerkstatt/geopilot-hosting/blob/main/.env.example#L88-L93) auch - in der Env verwalten und via `GetAuthOptions` (mit irendwelchen defaults) an den Client übergeben könnten. Was meint ihr @Philippluca @Stevic-Nenad ?